pandas/tests/io/json/test_normalize.py

```diff
@@ -153,6 +151,8 @@ def test_simple_records(self):
 
         tm.assert_frame_equal(result, expected)
 
+    # TODO(ArrayManager) sanitize S/U numpy dtypes to object
+    @td.skip_array_manager_not_yet_implemented
```

Comment:
Is this related to this change?

Reply:
It's related in the sense that this PR enables running all JSON tests for the ArrayManager case, except that a few tests (like the above one) are failing for other unrelated reasons, and thus need to be skipped. 
I did a precursor to do that first in https://github.com/pandas-dev/pandas/pull/41075 (and until that's merged, it's included here as the first commit)

Reply:
Merged the precursor, and rebased this PR now. So this change is no longer present in the diff.